vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/BiodataVariantToVariantContextConverterTest.java

```diff
@@ -400,6 +402,43 @@ public void twoStudiesConflictingNamesSingleVariant() {
         checkVariantContext(variantContext, CHR_1, 1000, 1000, "T", "G", variant.getSourceEntries(), true);
     }
 
+
+    @Test
+    public void csqAnnotation() {
+        // create variant
+        VariantSource variantSource = createTestVariantSource(STUDY_1);
+        String variantLine = String
+                .join("\t", CHR_1, "1000", "id", "C", "A", "100", "PASS", ".", "GT", "0|0", "0|0", "0|1", "1|1", "1|1",
+                      "0|1");
+        List<Variant> variants = variantFactory.create(variantSource, variantLine);
+        assertEquals(1, variants.size());
+        // populate variant with test csq data
+        variants.get(0).getAnnotation().setAlternativeAllele("aAllele");
+        List<ConsequenceType> consequenceTypes = new ArrayList<>();
+        List<String> soNames = new ArrayList<>(Arrays.asList("regulatory_region_ablation", "3_prime_UTR_variant"));
+        ConsequenceType consequenceType = new ConsequenceType("gene", "ensembleGeneId",
+                                                              "EnsembleTransId", "strand",
+                                                              "bioType", 10, 10, 10,
+                                                              "aaChange", "codon", new ArrayList<>(), soNames);
+        ConsequenceType consequenceType2 = new ConsequenceType("gene2", "ensembleGeneId2",
```

Comment:
It would be interesting to leave an empty attribute. Also, having different so terms in each annotation (two SO terms in one, to check is separating them using '&', and just one in the other)